src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4374,7 +4374,10 @@ enum Mounts
 
     // Big Blizzard Bear
     SPELL_BIG_BLIZZARD_BEAR_60          = 58997,
-    SPELL_BIG_BLIZZARD_BEAR_100         = 58999
+    SPELL_BIG_BLIZZARD_BEAR_100         = 58999,
+    SPELL_BIG_BLIZZARD_BEAR_150         = 58999,
+    SPELL_BIG_BLIZZARD_BEAR_280         = 58999,
+    SPELL_BIG_BLIZZARD_BEAR_310         = 58999
```

Comment:
Only 60 & 100 exist for this mount. No need to double the same spell Id here.

Reply:
I aggre, but it doesn't work :smiley: